src/calibration.c

```diff
@@ -107,27 +107,44 @@ void run_static_turn_right_profile(void)
 void run_front_sensors_calibration(void)
 {
 	float linear_acceleration = get_linear_acceleration();
+	float linear_deceleration = get_linear_deceleration();
+	float required_deceleration;
 	int32_t target_micrometers;
 	int32_t micrometers_to_stop;
+	uint32_t ticks_to_stop;
 
 	disable_walls_control();
+	side_sensors_calibration();
 	enable_motor_control();
 
-	set_linear_acceleration(4.);
+	set_linear_acceleration(3.);
 
-	target_micrometers = get_encoder_average_micrometers() +
-			     1.3 * CELL_DIMENSION * MICROMETERS_PER_METER;
+	target_micrometers =
+	    get_encoder_average_micrometers() +
+	    (2 * CELL_DIMENSION - MOUSE_TAIL - MOUSE_HEAD - WALL_WIDTH) *
+		MICROMETERS_PER_METER;
 	set_target_angular_speed(0.);
 	set_target_linear_speed(.3);
-	micrometers_to_stop = (int32_t)required_micrometers_to_speed(0.);
+	micrometers_to_stop = 15000;
 	while (get_encoder_average_micrometers() <
-	       target_micrometers - micrometers_to_stop)
+	       target_micrometers - micrometers_to_stop) {
 		log_front_sensors_calibration();
-	set_target_angular_speed(0.);
+		sleep_us(1000);
+	}
+	required_deceleration =
+	    (.3 * .3) /
```

Comment:
Macro for set_target_linear_speed and these .3* .3.

Reply:
Created a variable to store that value (58693b9e4a6448637cbbdbccf30c05b4fb6d4dd8).